pyproject.toml

```diff
@@ -126,7 +126,7 @@ pyOpenSSL = ">=16.0.0"
 PyYAML = ">=3.11"
 pyasn1 = ">=0.1.9"
 pyasn1-modules = ">=0.0.7"
-bcrypt = ">=3.1.0"
+bcrypt = ">=3.1.7"
```

Comment:
Sorry, meant to post this as a comment with the review:

Related to setuptools somehow? https://github.com/pyca/bcrypt#317

Have we given packagers a heads-up?